api/tests/core/offerers/test_api.py

```diff
@@ -1403,6 +1405,12 @@ def assert_common_action_history_extra_data(self, action: history_models.ActionH
         assert action.extraData["venue_type_code"] == offerers_models.VenueTypeCode.MOVIE.name
         assert action.extraData["web_presence"] == "www.example.com, instagram.com/example, @example@mastodon.example"
 
+    def assert_only_welcome_email_to_pro_was_sent(self) -> None:
```

Comment:
```suggestion
    def assert_only_welcome_to_pro_email_was_sent(self) -> None:
```
C'est un détail, fais comme tu le sens